src/hooks/useBestTrade.ts

```diff
@@ -14,6 +15,9 @@ import useIsWindowVisible from './useIsWindowVisible'
 // Prevents excessive quote requests between keystrokes.
 const DEBOUNCE_TIME = 350
 
+// Temporary until we remove the feature flag.
```

Comment:
if there is a task tracking this experiment, it may help to make this a TODO

`TODO (WEB-1234): clean up this experiment when finished`

Reply:
i think it's fine without bc typescript will complain if i remove the flag but leave the variable